sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -144,6 +144,7 @@ class StatisticsSuite extends QueryTest with BeforeAndAfterAll {
         expectedAnswer: Seq[Row],
         ct: ClassTag[_]) = {
       before()
+      conf.setConf("spark.sql.autoSortMergeJoin", "false")
```

Comment:
Because I was using true as default value for comlete join tests and forget to reset it back :p